pwndbg/arch.py

```diff
@@ -29,6 +30,21 @@ def fix_arch(arch):
     if 'arm' in arch:
         return 'armcm' if '-m' in arch else 'arm'
 
+    # handle some corner cases
```

Comment:
what's the corner case? when does it occur?

Reply:
This is because the definitions of "architecture" are different between gdb and pwndbg. We handle some "translation" here. for example. i386:x86-64 in gdb should be treated as x86-64 in pwndbg

Reply:
Can we mention this in a comment?